webrtc.html

```diff
@@ -4166,7 +4166,7 @@ <h2 id="sec.stats-model">Statistics Model</h2>
 
       <dl class="idl" title="partial interface RTCPeerConnection">
         <dt>Promise&lt;RTCStatsReport&gt; getStats(
-          optional MediaStreamTrack? selector)</dt>
+          optional MediaStreamTrack? selector = null)</dt>
```

Comment:
I'm not sure what the correct syntax is here, but it seems to me that "optional MediaStreamTrack? selector = null" makes the selector optional in 3 different ways. If it has a default value, it doesn't need to be optional, does it?
I suppose either

optional MediaStreamTrack selector

or 

MediaStreamTrack? selector = null

would have the same effect, but stacking them seems strange.